DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -223,68 +196,67 @@ function Set-TargetResource
     Update-LocationTagInApplicationHostConfigForAuthentication -WebSite $EndpointName -Authentication "basic"
     Update-LocationTagInApplicationHostConfigForAuthentication -WebSite $EndpointName -Authentication "windows"
         
-
     if ($IsBlue)
     {
         Write-Verbose "Set values into the web.config that define the repository for BLUE OS"
```

Comment:
OLE does not work on Windows 2012 R2 Core . Was not the original code for setting up eseprovider correct for blue or I am missing something?
Should not there be only two checks?
 1. Blue and above 
 2. Everything else ?